db/migrate/20160318133352_add_deprecated_to_vms.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeprecatedToVms < ActiveRecord::Migration[5.0]
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def change
+    add_column :vms, :deprecated, :boolean, :default => false
```

Comment:
Remove the default value.  Since the default value for a boolean is nil by default, then Rails knows that's falsey anyway, so it's not needed.